images/base/Dockerfile

```diff
@@ -74,18 +74,15 @@ RUN clean-install \
 COPY 10-network-security.conf /etc/sysctl.d/
 
 # Install containerd and runc binaries from kind-ci/containerd-nightlies repository
-# The repository contains latest stable releases and nightlies using golang's pseudo-version
-# CONTAINERD_VERSION=0.0.0-yyyymmddhhmmss-commitid
-ARG CONTAINERD_VERSION="0.0.0-20191007143651-efd38f48"
-ARG CONTAINERD_BASE_URL="https://github.com/kind-ci/containerd-nightlies/releases/download/"
+# The repository contains latest stable releases and nightlies built for multiple architectures
+ARG CONTAINERD_VERSION="v1.3.0-7-g0b43a311"
```

Comment:
should the `v` be part of the version given it's replaced in all references bellow?

Reply:
`v` is part of the version, however for :reasons: it's not in the download URL currently :roll_eyes:

Reply:
this is also true of the official releases

Reply:
i see, so at some point it might become part of the URLs.

Reply:
i'm not really sure what makes more sense. multiple formats are used, but this is the format used for VERSION in the build tooling, in the CLI, and in the tags. if they become a go module then the v will become more present.

https://github.com/containerd/containerd/releases/download/v1.3.0/containerd-1.3.0.linux-amd64.tar.gz

Reply:
versus ours:
https://github.com/kind-ci/containerd-nightlies/releases/download/containerd-1.3.0-7-g0b43a311/containerd-1.3.0-7-g0b43a311.linux-amd64.tar.gz

(which are still a work in progress.. we're actively improving that infra eg https://github.com/kind-ci/containerd-nightlies/pull/8)

Reply:
i think i prefer to have the `v` everywhere. :thinking:

Reply:
+1 cc @aojea